### PR TITLE
fix: remove unused code (Navigator) [WPB-6908]

### PR DESCRIPTION
**Pull request by caldrian**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-6908" title="WPB-6908" target="_blank">BugWPB-6908</a>  [iOS] Self user shown as verified when certificate is expired
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Removes some unused code.

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: wire-ios/Wire-iOS/Sources/AppDelegate.swift [hunk 1]
```diff
@@ -268,11 +268,11 @@ private extension AppDelegate {
             fatalError("sessionManager is not created")
         }
 
-        let navigator = Navigator(NoBackTitleNavigationController())
-        appRootRouter = AppRootRouter(viewController: viewController,
-                                      navigator: navigator,
-                                      sessionManager: sessionManager,
-                                      appStateCalculator: appStateCalculator)
+        appRootRouter = AppRootRouter(
+            viewController: viewController,
```
Comment (netbe) [inline wire-ios/Wire-iOS/Sources/AppDelegate.swift:272]: not sure we keep the functonality to have no back button title here

**Review by netbe — CHANGES_REQUESTED** [review]
can target develop

**Comment by markusfassbender** [comment]
@caldrian @netbe I think now there are more commits to `develop` than we want. Probably you need to make a new branch starting from `develop` and cherry pick your commits.

File: wire-ios/Wire-iOS/Sources/AppDelegate.swift (hunk 1 above)
Comment (caldrian) [reply wire-ios/Wire-iOS/Sources/AppDelegate.swift:272]: I'm not sure what you mean. If I'm not mistaken, I just removed dead code.

Comment (netbe) [reply wire-ios/Wire-iOS/Sources/AppDelegate.swift:272]: I think you're removing the `NoBackTitleNavigationController` which is a subclass of UINavigationController  that would hide the title in back button.

Comment (netbe) [reply wire-ios/Wire-iOS/Sources/AppDelegate.swift:272]: ok bye bye deadcode

**Review by netbe — APPROVED** [review]

**Review by markusfassbender — APPROVED** [review]

**Review by David-Henner — APPROVED** [review]

**Pull request merged**